presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1517,6 +1517,32 @@ public void testGroupingCombinationsDistinct()
     }
 
     @Test
+    public void testRollupOverUnion()
+            throws Exception
+    {
+        MaterializedResult actual = computeActual("" +
+                "SELECT\n" +
```

Comment:
This SELECT is short, so putting the values on the same line makes it easier to read